client/src/components/HomePage/Registration.jsx

```diff
@@ -57,13 +94,13 @@ class Registration extends Component {
 
   render() {
     return (
-      <div className="white">
+      <div className="white" id="registration-container">
```

Comment:
Why did you add this id?

Reply:
Now you don't have css for this. `All elements must have their id`, who told you this?

Reply:
I can delete this. But I dont see a problem for give id. Or just proof that are wrong.

Reply:
The first [link](https://css-tricks.com/the-difference-between-id-and-class/#article-header-id-9) in google. You can find more, but "If you don't need them, don't use them".
The [article](https://dev.to/claireparker/reasons-not-to-use-ids-in-css-4ni4) why `class` is better than `id` in css. If you need css - use class, if you need js - use id.